### PR TITLE
Release of XWiki 9.9

### PR DESCRIPTION
**Pull request by evalica**

### CODE REVIEW

**Comment by tianon** [comment]
FYI @vmassol

<details>
<summary>Diff:</summary>

```diff
diff --git a/xwiki_stable-postgres/Dockerfile b/xwiki_stable-postgres/Dockerfile
index 1dd58f2..b10375e 100644
--- a/xwiki_stable-postgres/Dockerfile
+++ b/xwiki_stable-postgres/Dockerfile
@@ -44,9 +44,9 @@ RUN apt-get update && \
 # Install XWiki as the ROOT webapp context in Tomcat
 # Create the Tomcat temporary directory
 # Configure the XWiki permanent directory
-ENV XWIKI_VERSION=9.8.1
+ENV XWIKI_VERSION=9.9
 ENV XWIKI_URL_PREFIX "http://maven.xwiki.org/releases/org/xwiki/platform/xwiki-platform-distribution-war/${XWIKI_VERSION}"
-ENV XWIKI_DOWNLOAD_SHA256 26f55d585ed1b05cf730af585ab1cd0751e2b09770e2ab27c11e3b55027b9733
+ENV XWIKI_DOWNLOAD_SHA256 6141614d0cebcdec83b83642e94b0b8b70ed832e9b32db2c34c31549e546a3a9
 RUN rm -rf /usr/local/tomcat/webapps/* && \
   mkdir -p /usr/local/tomcat/temp && \
   mkdir -p /usr/local/xwiki/data && \
diff --git a/xwiki_stable/Dockerfile b/xwiki_stable/Dockerfile
index 1fe79a1..013a950 100644
--- a/xwiki_stable/Dockerfile
+++ b/xwiki_stable/Dockerfile
@@ -44,9 +44,9 @@ RUN apt-get update && \
 # Install XWiki as the ROOT webapp context in Tomcat
 # Create the Tomcat temporary directory
 # Configure the XWiki permanent directory
-ENV XWIKI_VERSION=9.8.1
+ENV XWIKI_VERSION=9.9
 ENV XWIKI_URL_PREFIX "http://maven.xwiki.org/releases/org/xwiki/platform/xwiki-platform-distribution-war/${XWIKI_VERSION}"
-ENV XWIKI_DOWNLOAD_SHA256 26f55d585ed1b05cf730af585ab1cd0751e2b09770e2ab27c11e3b55027b9733
+ENV XWIKI_DOWNLOAD_SHA256 6141614d0cebcdec83b83642e94b0b8b70ed832e9b32db2c34c31549e546a3a9
 RUN rm -rf /usr/local/tomcat/webapps/* && \
   mkdir -p /usr/local/tomcat/temp && \
   mkdir -p /usr/local/xwiki/data && \
````

</details>

**Comment by vmassol** [comment]
Thanks @tianon Please hold on since I believe the diff shows something wrong

File: library/xwiki [hunk 1]
```diff
@@ -6,13 +6,13 @@ GitCommit: fbc0fe3cbc07119c45ae8ea97c7e021d41e89b0a
 Directory: 8/mysql-tomcat
 
 Tags: 9, 9.8, 9.8.1, 9-mysql-tomcat, stable-mysql-tomcat, stable-mysql, stable
-GitCommit: 088aaf39de290ece849eabace996b21f3b99d954
+GitCommit: ed8536e2861198c2052f54634ada97723239f5e6
 Directory: 9/mysql-tomcat
```
Comment (vmassol) [inline library/xwiki]: This is not correct @evalica 

We only support LTS + the latest stable, so it should be:

```
Tags: 9, 9.9, 9-mysql-tomcat, stable-mysql-tomcat, stable-mysql, stable
GitCommit: ed8536e2861198c2052f54634ada97723239f5e6
```

File: library/xwiki [hunk 2]
```diff
@@ -6,13 +6,13 @@ GitCommit: fbc0fe3cbc07119c45ae8ea97c7e021d41e89b0a
...
 
 Tags: 8-postgres-tomcat, 8.4-postgres-tomcat, 8.4.5-postgres-tomcat, postgres-tomcat, lts-postgres-tomcat, lts-postgres
 GitCommit: fbc0fe3cbc07119c45ae8ea97c7e021d41e89b0a
 Directory: 8/postgres-tomcat
 
 Tags: 9-postgres-tomcat, 9.8-postgres-tomcat, 9.8.1-postgres-tomcat, stable-postgres-tomcat, stable-postgres
-GitCommit: 088aaf39de290ece849eabace996b21f3b99d954
+GitCommit: ed8536e2861198c2052f54634ada97723239f5e6
```
Comment (vmassol) [inline library/xwiki]: Same here

**Comment by vmassol** [comment]
ok it looks good now, thanks @evalica for fixing quickly :)

**Comment by tianon** [comment]
<details>
<summary>Diff:</summary>

```diff
diff --git a/_bashbrew-list b/_bashbrew-list
index 88bc8af..e179600 100644
--- a/_bashbrew-list
+++ b/_bashbrew-list
@@ -8,10 +8,8 @@ xwiki:8.4.5-postgres-tomcat
 xwiki:9
 xwiki:9-mysql-tomcat
 xwiki:9-postgres-tomcat
-xwiki:9.8
-xwiki:9.8-postgres-tomcat
-xwiki:9.8.1
-xwiki:9.8.1-postgres-tomcat
+xwiki:9.9
+xwiki:9.9-postgres-tomcat
 xwiki:latest
 xwiki:lts
 xwiki:lts-mysql
diff --git a/xwiki_stable-postgres/Dockerfile b/xwiki_stable-postgres/Dockerfile
index 1dd58f2..b10375e 100644
--- a/xwiki_stable-postgres/Dockerfile
+++ b/xwiki_stable-postgres/Dockerfile
@@ -44,9 +44,9 @@ RUN apt-get update && \
 # Install XWiki as the ROOT webapp context in Tomcat
 # Create the Tomcat temporary directory
 # Configure the XWiki permanent directory
-ENV XWIKI_VERSION=9.8.1
+ENV XWIKI_VERSION=9.9
 ENV XWIKI_URL_PREFIX "http://maven.xwiki.org/releases/org/xwiki/platform/xwiki-platform-distribution-war/${XWIKI_VERSION}"
-ENV XWIKI_DOWNLOAD_SHA256 26f55d585ed1b05cf730af585ab1cd0751e2b09770e2ab27c11e3b55027b9733
+ENV XWIKI_DOWNLOAD_SHA256 6141614d0cebcdec83b83642e94b0b8b70ed832e9b32db2c34c31549e546a3a9
 RUN rm -rf /usr/local/tomcat/webapps/* && \
   mkdir -p /usr/local/tomcat/temp && \
   mkdir -p /usr/local/xwiki/data && \
diff --git a/xwiki_stable/Dockerfile b/xwiki_stable/Dockerfile
index 1fe79a1..013a950 100644
--- a/xwiki_stable/Dockerfile
+++ b/xwiki_stable/Dockerfile
@@ -44,9 +44,9 @@ RUN apt-get update && \
 # Install XWiki as the ROOT webapp context in Tomcat
 # Create the Tomcat temporary directory
 # Configure the XWiki permanent directory
-ENV XWIKI_VERSION=9.8.1
+ENV XWIKI_VERSION=9.9
 ENV XWIKI_URL_PREFIX "http://maven.xwiki.org/releases/org/xwiki/platform/xwiki-platform-distribution-war/${XWIKI_VERSION}"
-ENV XWIKI_DOWNLOAD_SHA256 26f55d585ed1b05cf730af585ab1cd0751e2b09770e2ab27c11e3b55027b9733
+ENV XWIKI_DOWNLOAD_SHA256 6141614d0cebcdec83b83642e94b0b8b70ed832e9b32db2c34c31549e546a3a9
 RUN rm -rf /usr/local/tomcat/webapps/* && \
   mkdir -p /usr/local/tomcat/temp && \
   mkdir -p /usr/local/xwiki/data && \
````

</details>

**Comment by tianon** [comment]
LGTM

Build test of #3614; 43ed3c81 (`xwiki`):

```console

$ bashbrew build xwiki:8
Using bashbrew/cache:449a9a71fd0314fc83536c7c9fd5c96fb81fe44661c87933477f390ef11ce2d1 (xwiki:8)
Tagging xwiki:8
Tagging xwiki:8.4
Tagging xwiki:8.4.5
Tagging xwiki:8-mysql-tomcat
Tagging xwiki:mysql-tomcat
Tagging xwiki:lts-mysql-tomcat
Tagging xwiki:lts-mysql
Tagging xwiki:lts
Tagging xwiki:latest

$ test/run.sh xwiki:8
testing xwiki:8
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build xwiki:9
Using bashbrew/cache:e7475a861713069a5bfbf8e60681ccaebf74bd12f7bd5537612eac80ea2717d3 (xwiki:9)
Tagging xwiki:9
Tagging xwiki:9.9
Tagging xwiki:9-mysql-tomcat
Tagging xwiki:stable-mysql-tomcat
Tagging xwiki:stable-mysql
Tagging xwiki:stable

$ test/run.sh xwiki:9
testing xwiki:9
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build xwiki:8-postgres-tomcat
Using bashbrew/cache:69c63ace470d6aec98e746e2bba7e842bd87117dca07b4b5335d8c37b8f1187e (xwiki:8-postgres-tomcat)
Tagging xwiki:8-postgres-tomcat
Tagging xwiki:8.4-postgres-tomcat
Tagging xwiki:8.4.5-postgres-tomcat
Tagging xwiki:postgres-tomcat
Tagging xwiki:lts-postgres-tomcat
Tagging xwiki:lts-postgres

$ test/run.sh xwiki:8-postgres-tomcat
testing xwiki:8-postgres-tomcat
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build xwiki:9-postgres-tomcat
Using bashbrew/cache:303ac0064e7aef3c693ce97f56ee5756bd2b912287f11cf223218d4080f22e24 (xwiki:9-postgres-tomcat)
Tagging xwiki:9-postgres-tomcat
Tagging xwiki:9.9-postgres-tomcat
Tagging xwiki:stable-postgres-tomcat
Tagging xwiki:stable-postgres

$ test/run.sh xwiki:9-postgres-tomcat
testing xwiki:9-postgres-tomcat
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed

```

**Pull request merged**